index.html

```diff
@@ -1,16 +1,135 @@
+               <div class="col-sm-4">
+                   <h1 class="introduction text-center display-4">
+                       What is Lorem Ipsum?
+                   </h1>
+                   <p>
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy t
+                        ext ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived 
+                        not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s 
+                        with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus 
+                        PageMaker including versions of Lorem Ipsum.
+                        <br>
+                        <strong><i>Why do we use it?</i></strong>
+                        It is a long established fact that a reader will be distracted by the readable 
+                        content of a page when looking at its layout. The point of using Lorem Ipsum is 
+                        that it has a more-or-less normal distribution of letters, as opposed to using 
+                        'Content here, content here', making it look like readable English. Many desktop 
+                        publishing packages and web page editors now use Lorem Ipsum as their default 
+                        model text, and a search for 'lorem ipsum' will uncover many web sites still in 
+                        their infancy. Various versions have evolved over the years, sometimes by accident, 
+                        sometimes on purpose (injected humour and the like).
+                   </p>
+               </div>
+                <div class="col-sm-8">
+                   <img src="/images/pexels-vidal-balielo-jr-1682474.jpg" alt="image" 
+                      class="img-fluid img-thumbnail max-width: 100% height:auto"> 
+
+                   <div class="row" id="pic-div">
+                        <div class="col-sm-2">
+                            <img src="/images/20200719_164740.jpg" alt="" class="img-fluid img-thumbnail max-width: 100% height:auto">
+                        </div>
+                        <div class="col-sm-2">
+                            <img src="/images/20200805_131129.jpg" alt="" class="img-fluid img-thumbnail max-width: 100% height:auto">
+                        </div>
+                        <div class="col-sm-2">
+                            <img src="images/ulysse-pointcheval-1WmlAiYgnoI-unsplash.jpg" alt="" class="img-fluid img-thumbnail max-width: 100% height:auto">
+                        </div>
+                        <div class="col-sm-2">
+                            <img src="/images/20200706_171336.jpg" alt="" class="img-fluid img-thumbnail max-width: 100% height:auto">
```

Comment:
You cannot use these styling properties in a class value. max-width: 100% height:auto won't work. They need to be either declared inline like this: style="max-width: 100% height:auto" or better still in the style.css stylesheet.

Reply:
Also some of the image src are declared absolute from the route. I would remove the first '/' from each of these source attributes